spec/models/payment_providers/adyen_provider_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PaymentProviders::AdyenProvider, type: :model do
+  let(:provider) { FactoryBot.build_stubbed(:adyen_provider) }
```

Comment:
To fix the rubocop comments, ou should define this one as spec subject `subject(:provider)` and replace all other ocurrence of `subject` with a dedicated `let`.